### PR TITLE
Refer A Friend - reward & campaign updates

### PR DESCRIPTION
**Pull request by mendelB**
### What does this PR do?

This PR updates the Refer a Friend feature to be tailored for the new scholarship entries reward, and limited to the Teens for Jeans Campaign.

Mostly just asset and copy updates. I'll note some notable commits:
- https://github.com/DoSomething/phoenix-next/commit/0483ed662ad628c3ee7c50c1c0221ae646f2333a initially, the new assets had different dimensions, which led me to need to finesse the styling to adjust for the new dimensions. We ended up getting new images with the same ole dimensions, but since I'd already spent a bunch of time finicking, it felt nice to apply it here as well so that the images match the OG mocks a bit closer for [large](https://user-images.githubusercontent.com/12417657/69657858-1b021600-1049-11ea-8edc-11f8114f6fa1.png), [small](https://user-images.githubusercontent.com/12417657/69657984-556bb300-1049-11ea-8d7d-ece1fb30b3eb.png), and [extra small](https://user-images.githubusercontent.com/12417657/69657985-556bb300-1049-11ea-8a8c-b360453a8381.png) screens.
- https://github.com/DoSomething/phoenix-next/commit/5acbe637ad5a1c3c0333aa61f4a482888a679d37 and https://github.com/DoSomething/phoenix-next/commit/2a4dbbd02fab088d72b2635b5244967e12234e8f filter for the Teens For Jeans campaign (and it's dev sandbox counterpart), since that's what we're limiting this RAF run to.

### Any background context you want to provide?
We're changing the referral reward to scholarship entries, and limiting the initiative to the upcoming Teens for Jeans campaign.

We're still missing one asset swap for the [share link metadata](https://github.com/DoSomething/phoenix-next/blob/fef37af48a294cd906f0e3c81779c4e96caed0f6/app/Http/Controllers/ReferralPageController.php#L41). I'm just [waiting on that asset](https://www.pivotaltracker.com/story/show/169376190/comments/209129642)

### What are the relevant tickets/cards?

Refs [Pivotal ID #169376190](https://www.pivotaltracker.com/story/show/169376190)

### Checklist

* [ ] Documentation added for new features/changed endpoints.
* [ ] Added appropriate feature/unit tests.
* [ ] Added screenshot to PR description of related front-end updates on **small** screens.
* [ ] Added screenshot to PR description of related front-end updates on **medium** screens.
* [ ] Added screenshot to PR description of related front-end updates on **large** screens.

I'll add the PR deploy to the auth whitelist, and paste some relevant links rather than screenshot the various copy and asset changes.

[Teens for Jeans! for the referral affirmation](https://dosomething-phoenix-de-pr-1774.herokuapp.com/us/campaigns/test-teens-for-jeans/action)

[Alpha page](https://dosomething-phoenix-de-pr-1774.herokuapp.com/us/refer-friends?campaign_id=9001)

[Beta page](https://dosomething-phoenix-de-pr-1774.herokuapp.com/us/join?user_id=5bbd00271f1c570099415d02&campaign_id=9001)

### CODE REVIEW

File: app/Http/Controllers/ReferralPageController.php [hunk 1]
```diff
@@ -27,7 +27,7 @@ public function show(Request $request)
 
         $firstName = $user->first_name;
         $title = 'Do Something Good With '.$firstName.'!';
-        $callToAction = 'Make an impact with '.$firstName.' by completing one of DoSomething\'s volunteer campaigns. (You\'ll both receive a $5 gift card!)';
+        $callToAction = 'Make an impact with '.$firstName.' by completing one of DoSomething\'s volunteer campaigns. (You\'ll both double your chances of winning the campaign scholarship!)';
```
Comment (katiecrane) [inline app/Http/Controllers/ReferralPageController.php]: I have a question about this wording. So you get one entry, then you invite a friend and you now have 2 entries. If you invite another friend, do you now have 4 entries?

Comment (mendelB) [reply app/Http/Controllers/ReferralPageController.php]: Ah yeah, that math is off! I guess we're kinda implying that your friend still only has one entry (maybe making you feel more special as their chief referral lol)

**Comment by mendelB** [comment]
Updated the PR description with some links for testing:
[Teens for Jeans! for the referral affirmation](https://dosomething-phoenix-de-pr-1774.herokuapp.com/us/campaigns/test-teens-for-jeans/action)

[Alpha page](https://dosomething-phoenix-de-pr-1774.herokuapp.com/us/refer-friends?campaign_id=9001)

[Beta page](https://dosomething-phoenix-de-pr-1774.herokuapp.com/us/join?user_id=5bbd00271f1c570099415d02&campaign_id=9001)

File: resources/assets/components/pages/ReferralPage/Beta/BetaPage.js [hunk 2]
```diff
@@ -16,26 +17,27 @@ const REFERRAL_PAGE_USER = gql`
...
   if (!userId) {
-    return <ErrorBlock />;
+    return <ErrorPage />;
   }
 
-  const campaignId = query('campaign_id');
-  const displayPrimaryCampaign =
-    campaignId && campaignId !== SECONDARY_CAMPAIGN_ID;
+  // Defaults to Teens For Jeans campaign.
+  const DEFAULT_CAMPAIGN_ID = '9037';
```
Comment (aaronschachter) [inline resources/assets/components/pages/ReferralPage/Beta/BetaPage.js]: What do you think of defining this as `DEFAULT_REFERRAL_CAMPAIGN_ID` over in constants?

Comment (mendelB) [reply resources/assets/components/pages/ReferralPage/Beta/BetaPage.js]: Ah yes, that makes good sense!

**Pull request merged**